### ISSUE TITLE
[Adaptive Cards iOS - WeatherCompact]: Temperature '46' should be more descriptive under 'WeatherCompact' screen.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7243]
https://microsoft.visualstudio.com/OS/_workitems/edit/33765765
